src/components/pages/About/AboutUs.js

```diff
@@ -35,14 +35,14 @@ export default function AboutUs() {
                     target="_blank"
                     rel="noopener noreferrer"
                   >
-                    <i class="fab fa-github-square fa-3x" />
+                    <i className="fab fa-github-square fa-3x" />
```

Comment:
I wasn't sure if the icons would work still with className instead of class. you made a great tweak AND I learned something. Thank you